docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Gatsby Docs Translation Guide
+---
+
+Since the [RFC on localization](https://github.com/gatsbyjs/rfcs/pull/42) for Gatsby, there is now an ongoing effort to translate the content on Gatsbyjs.org into multiple languages. For members of the community around the world building Gatsby sites and learning about web development, having docs and learning materials in your own language provides a lot of value. There's also a great opportunity for folks like you to contribute to the translations of the Gatsby docs.
+
+If you're a native speaker of a language other than English and you're interested in helping translate gatsbyjs.org, here is some information for you to know:
+
+## The general process
+
+Each translation has its own repository in the [gatsbyjs](https://github.com/gatsbyjs/) organization on GitHub, with designated codeowners to review and approve changes. This doc will outline how to request a new language repository with its own team, what decisions each team can make, and suggestions for how you can work together.
+
+The Gatsby learning team is in charge of determining priorities for which docs should be translated. Refer to the [i18n page spreadsheet](https://docs.google.com/spreadsheets/d/1u2amGnqFLKxJuL5h9UrDblUueFgg0EBt7xbau4n8iTM/edit) to get the most up-to-date priority list.
+
+## Use English as the source
+
+The [gatsbyjs.org](https://gatsbyjs.org) website is written in English and should be considered the source material for all translations (as opposed to translating from another translation). When a repository is created, it will provide the space to copy over prioritized docs and [make pull requests](/contributing/how-to-open-a-pull-request/) against them in the relevant language.
+
+## Process for starting a new translation
+
+The first step for starting a new translation is to check what exists. So far, there are repositories for the following languages:
+
+- [Spanish](https://github.com/gatsbyjs/gatsby-es)
+- [Russian](https://github.com/gatsbyjs/gatsby-ru)
+
+You can find the list of all translation requests here: https://github.com/gatsbyjs/gatsby/issues?utf8=%E2%9C%93&q=is%3Aissue+%22New+Translation+Request%22+
+
+> Note: Once a new translation repository is created, feel free to add it here in a PR!
```

Comment:
This should be replaced with the /languages page on the website once that's actually created.

Reply:
Linking to the repos might be nice for contributing purposes, but it would definitely be harder to maintain...that would be easier.